test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/ViewModels/ReadmePreviewViewModelTests.cs

```diff
@@ -61,22 +63,66 @@ public async Task LoadReadmeAsync_WithoutReadmeUrl_NoReadmeReturned(string readm
         }
 
         [Fact]
-        public async Task LoadReadmeAsync_WithReadmeUrl_ReadmeReturned()
+        public async Task LoadReadmeAsync_WithLocalReadmeUrl_RenderLocalReadmeTrue_ReadmeReturned()
         {
             //Arrange
             var readmeContents = "readme contents";
             using Stream stream = new MemoryStream(Encoding.UTF8.GetBytes(readmeContents));
             var mockFileService = new Mock<INuGetPackageFileService>();
             mockFileService.Setup(x => x.GetReadmeAsync(It.IsAny<Uri>(), It.IsAny<CancellationToken>())).ReturnsAsync(stream);
             var target = new ReadmePreviewViewModel(mockFileService.Object);
+            var package = new DetailedPackageMetadata();
+            package.ReadmeFileUrl = "C://path/to/readme.md";
 
             //Act
-            await target.LoadReadmeAsync("C://path/to/readme.md", CancellationToken.None);
+            await target.LoadReadmeAsync(package, true, CancellationToken.None);
+
 
             //Assert
             Assert.False(target.ErrorLoadingReadme);
             Assert.Equal(readmeContents, target.ReadmeMarkdown);
             Assert.True(target.CanDetermineReadmeDefined);
         }
+
+        [Fact]
+        public async Task LoadReadmeAsync_WithLocalReadmeUrl_RenderLocalReadmeFalse_ReadmeReturned()
```

Comment:
The test checks for no readme content. Is the test name wrong or is the assertion wrong?

```suggestion
        public async Task LoadReadmeAsync_WithLocalReadmeUrl_RenderLocalReadmeFalse_NoReadmeReturned()
```

Reply:
I updated the tests I think this has been resolved.